exercise_2/part_7.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Create a table</title>
+    <style type="text/css">
+      body {margin: 30px;}
+    </style>
+  </head>
+  <body>
+    <table id="myTable" border="1">
+    </table>
+    <form>
+      <input type="button" onclick="createTable()" value="Create the table">
+    </form>
+    <script>
+      function createTable() {
+        const rowNew = prompt("Input rows: ");
+        const columnNew = prompt("Input rows: ");
+        for(let row = 0; row<parseInt(rowNew,10); row++){
```

Comment:
space. `row < parseInt(rowNew,10)`. If you have 10 rows, your loop will call parseInt function 10 times. Please create a variable for it